internal/serve/serve.go

```diff
@@ -90,6 +91,10 @@ type ServeOptions struct {
 	EnableMultiTenantDB             bool
 	tenantManager                   tenant.ManagerInterface
 	tenantRouter                    db.DataSourceRouter
+	EventProducer                   events.Producer
+	Brokers                         []string
+	Topics                          []string
+	ConsumerGroupID                 string
```

Comment:
Does the server need anything Kafka-related? If not, we shouldn't be adding these fields to its object.

If it does need Kafka, we could be injecting a Kafka object, rather than injecting a few fields that would need to be tied together later. WDYT?

Reply:
I don't think we need it! I'm going to remove it.